### PR TITLE
Addressed feedback about Flex and FlexItem components

### PR DESCRIPTION
**Pull request by mperrotti**

### CODE REVIEW

**Comment by sadafie** [comment]
We can talk more about this/workflow in today's meeting, but is this intentionally merging back into the `responsify` branch? Any reason why it can't just go into `master`?

**Comment by mperrotti** [comment]
@sadafie: I'm intentionally merging back into `responsify` because that's where I was told I should submit my PR. I'm happy to make a PR in `master` if we decide that makes more sense

**Comment by sadafie** [comment]
@mperrotti okie dokie. I probably just missed that conversation, but let's go over later in the meeting. at the very least just for my benefit. haha thanks!

**Comment by mmcgahan** [comment]
this looks awesome!

File: assets/scss/flex.scss [hunk 1]
```diff
@@ -212,6 +193,17 @@ Class                   | Variants
 
 }
 
+/*
+Content justification variants are also available (see also: <a class="link" target="_blank" href="https://css-tricks.com/almanac/properties/j/justify-content/">css tricks flexbox demo</a>)
+Using this signature: `.at[Breakpoint]_flex--[justification]`
+
+Class                   | Variants
+----------------------- | -------------------------------
+`.flex--flexEnd`        | justifies content to `flex-end`
```
Comment (akdetrick) [inline assets/scss/flex.scss]: should we be calling these `jusify--[variant]` for clarity? There are a lot of `flex` properties and these only pertain to `justify-content`

File: assets/scss/flex.scss [hunk 2]
```diff
@@ -220,6 +212,15 @@ Class                   | Variants
 
 }
 
+/*
+Alignment variants are also available using `align-items` (see also: <a class="link" target="_blank" href="https://css-tricks.com/almanac/properties/a/align-items/">css tricks flexbox demo</a>)
+
+Class                   | Variants
+----------------------- | -------------------------------
+`flex--alignTop`        | aligns content to `flex-start`
```
Comment (akdetrick) [inline assets/scss/flex.scss]: we can leave these classes and merge properties with existing `align--[variant]` classes later

File: assets/scss/flex.scss (hunk 1 above)
Comment (mperrotti) [inline assets/scss/flex.scss]: 👍

File: assets/scss/flex.scss (hunk 2 above)
Comment (mperrotti) [inline assets/scss/flex.scss]: Alright, I'll leave them in and file an issue to merge the two

**Pull request merged**